### ISSUE TITLE
Store rework in anticipation of families/corporations/shared residents

### ISSUE THREAD

**Issue by SatsukiShizuka** [StrangeLoopGames/EcoSuggestions#1339]
STORE SUGGESTION
With cohabitation incoming, there's a good chance people of different professions will be living and working out of shared spaces. While people can opt to share their inventories with roommates/family without compensation, others may find it a nightmare. One of the biggest problems I've had is the inability to use a shop on your own property (or temporarily given access to), until you've been unauthorized. This is because how the store is programmed - if you're authorized, the main page is a price-setter. If you're not, it becomes an actual shop screen.

Maybe these two things should be separated (storefront/store-back), so that one could shop at a store regardless. A little silly if you can shop at your own store, giving money to your own account, but it should in theory work too!
^ It should sound silly if "I buy using X wallet to the store which stores in X wallet" but should work. OTOH this would be absolutely a must if "I am the government, but I use the government account to buy from my business and would like a fair amount to be transferred" is to be setup!

**Comment by sirPattrick** [comment]
Yeah, likewise,  i personally made a bunch of Asphalt, and then I set up a Treasury funded store I couldn't sell my asphalt to the store.  I had to transfer funds out, which looks sus, if you just see the treasurer moving funds into their own pocket.
